packages/domain-search/src/ui/domain-search-controls/filter-popover-tld.tsx

```diff
@@ -0,0 +1,25 @@
+type Props = {
+	tld: string;
+	addTldToFilter: ( tld: string ) => void;
+};
+
+export const FilterPopoverTld = ( { tld, addTldToFilter }: Props ) => {
+	return (
+		<div
```

Comment:
Can't we use a button here instead of a div?

Reply:
Yep, I've updated the code